rust/connlib/snownet/src/node.rs

```diff
@@ -138,22 +139,27 @@ pub struct Node<T, TId, RId> {
     stats: NodeStats,
     buffer_pool: BufferPool<Vec<u8>>,
 
-    mode: T,
+    role: T,
     rng: StdRng,
+
+    /// The number of seconds since the UNIX epoch.
+    unix_ts: Duration,
+    /// The [`Instant`] at the time we read the UNIX epoch above.
+    unix_now: Instant,
```

Comment:
do we need both of these fields? as `unit_ts` is unused in this PR, I can't tell it's purpose

Reply:
We do need both, see https://github.com/firezone/boringtun/pull/126 for reference.

`unix_ts` are the number of seconds since the UNIX epoch. `unix_now` is the `Instant` at the same time we took `unix_ts`. The combination of the two allows us to compute the UNIX timestamp for a given `Instant`.